snapcraft.yaml

```diff
@@ -78,6 +78,7 @@ parts:
     - build-essential
     - cmake
     - curl
+    - dirmngr
```

Comment:
Is this the dependency that is missing on Fedora and not on Ubuntu?

Reply:
This needs to be installed for `linux_optional_compilation_dependencies.sh` to add the key associated with the ROS install. You could presumably add it from that script instead of adding it here.

Reply:
Ok. So it seems that this is installed by default on Ubuntu, because we don't have this issue when installing the Webots development system.
Yes, this should be added to the `linux_optional_compilation_dependencies.sh`.